org.eclipse.debug.ui/ui/org/eclipse/debug/internal/ui/SWTFactory.java

```diff
@@ -249,6 +249,27 @@ public static Label createLabel(Composite parent, String text, Font font, int hs
 		return l;
 	}
 
+	/**
+	 * Creates a new label widget
+	 *
+	 * @param parent  the parent composite to add this label widget to
+	 * @param text    the text for the label
+	 * @param hspan   the horizontal span to take up in the parent composite
+	 * @param gdSpace to grab the excess horizontal space of the Label
```

Comment:
This is a new API and the param gdSpace and it's javadoc is not very clear. Can be changed to `grabExcessHorizontalSpace/grabHorizontalSpace/grabHSpace - specifies whether the width of the label changes depending on the size of the parent Composite`. (modified from the javadoc of grabExcessHorizontalSpace)